tests/autoinstrumentation/tox.ini

```diff
@@ -0,0 +1,37 @@
+[tox]
+skipsdist = true
+envlist = py3{7,8,9}
+
+[testenv]
+sitepackages = True
+install_command = pip install {opts} {packages}
+
+deps =
+    -rrequirements.txt
+
+whitelist_externals =
+    pytest
+    sleep
+    docker-compose
+    test_flask_1.sh
+    bash
+    test_flask_2.sh
+
+setenv =
+  PYTHONPATH=../../src
+  HT_SERVICE_NAME=python_agent_test1
+  HT_LOG_LEVEL={env:HT_LOG_LEVEL:}
+  HT_ENABLE_CONSOLE_SPAN_EXPORTER=True
+
+commands =
+    docker-compose stop mysqldb
+    bash -ec "if [ `uname` = 'Linux' ] && [ "{env:GITHUB_ACTIONS:false}" != "true" ]; then chcon -h system_u:object_r:bin_t:s0 sql; chcon -Rt svirt_sandbox_file_t sql; fi"
```

Comment:
Could you please clarify what this do and why it is needed?

Reply:
Set selinux permissions on volumes being mounted into docker containers. Some of us have selinux enabled locally:).

The ubuntu VMs where the pipeline runs does not have selinux enabled. 

I have added a comment indicating this.